src/faults/AbstractPerturbation.jl

```diff
@@ -66,6 +66,8 @@ function simulate(np::AbstractPerturbation, powergrid::PowerGrid, x1, timespan;
 end
 
 simulate(np::AbstractPerturbation, op::State, timespan) = simulate(np, op.grid, op.vec, timespan)
+simulate(np::AbstractPerturbation, powergrid::PowerGrid,op::State, timespan) = simulate(np, powergrid, op.vec, timespan)
```

Comment:
@SabineAuer That was correct before. op has two fields, `grid::PowerGrid` and `vec::Array`.
There is no need to pass two grid instances.

Reply:
Anton and I were working on the Tecnalia Branch and realized this wasn't working. Now it does.

Reply:
what was the error? I used this all the time without problems. :-)